components/search/src/selector.rs

```diff
@@ -478,14 +462,9 @@ mod tests {
         );
 
         let mut result = Arc::clone(&selector).filter_engine_configuration(SearchUserEnvironment {
-            locale: "fi".into(),
-            region: "FR".into(),
-            update_channel: SearchUpdateChannel::Default,
             distribution_id: String::new(),
```

Comment:
Should `distribution_id` be removed? I noticed there no specific value set for it. 

It similar to `version: String::new()`  which has been removed.

Reply:
In the tests in this section, we're using `distribution_id` and `app_name` for the environment filtering parameters. Hence I wanted to keep them both there explicitly so that it was clearer that the `distribution_id` wasn't being set and that environment section wouldn't apply.